### PR TITLE
build/pkgs/python_flint: New

### PR DESCRIPTION
**Pull request by mkoeppe**
[python-flint](https://github.com/flintlib/python-flint) is an optional dependency of the upcoming SymPy 1.13 (https://groups.google.com/g/sympy/c/S3LL7hYvD_A) @oscarbenjamin

See also
- #36641

### :memo: Checklist

- [x] The title is concise, informative, and self-explanatory.
- [x] The description explains in detail what this PR is about.
- [ ] I have linked a relevant issue or discussion.
- [ ] I have created tests covering the changes.
- [ ] I have updated the documentation accordingly.

### :hourglass: Dependencies

- Depends on #37222 
- Depends on #36999 (merged here)

### CODE REVIEW

File: build/pkgs/python_flint/install-requires.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+python-flint
```
Comment (kwankyu) [inline build/pkgs/python_flint/install-requires.txt]: Do you want to change the filename to `version_requirements.txt` here in preparation for #36999?

Comment (mkoeppe) [reply build/pkgs/python_flint/install-requires.txt]: Good idea, done.

**Review by kwankyu — APPROVED** [review]
Thanks.

**Comment by mkoeppe** [comment]
Thank you!

**Pull request merged**